src/UuidFactoryInterface.php

```diff
@@ -179,4 +179,16 @@ public function fromDateTime(
         ?Hexadecimal $node = null,
         ?int $clockSeq = null
     ): UuidInterface;
+
+    /**
+     * Creates a UUID from a Hexadecimal object
+     *
+     * @param Hexadecimal $hex Hexadecimal object representing a hexadecimal number
+     *
+     * @return UuidInterface A UuidInterface instance created from the Hexadecimal 
+     * object representing a hexadecimal number
+     *
+     * @psalm-pure
+     */
+    public function fromHexadecimal(Hexadecimal $hex): UuidInterface;
```

Comment:
I'm unable to add this to `UuidFactoryInterface` until version 5, since it's a major breaking change to anyone who is implementing this interface.

Reply:
Makes sense. Will remove it from the interface.